dspace-api/src/main/java/org/dspace/content/service/DSpaceObjectService.java

```diff
@@ -202,8 +202,8 @@
      * @param values    the values to add.
      * @throws SQLException if database error
      */
-    public void addMetadata(Context context, T dso, String schema, String element, String qualifier, String lang,
-                            List<String> values) throws SQLException;
+    public List<MetadataValue> addMetadata(Context context, T dso, String schema, String element, String qualifier,
```

Comment:
Could you update the JavaDocs for all these modified methods to have a `@return` annotation now & describe what is returned?

Reply:
The JavaDocs have been created